spec/requests/api/service_catalogs_spec.rb

```diff
@@ -390,6 +390,7 @@ def init_st(service_template, resource_action)
 
     it "supports single order request" do
       api_basic_authorize subcollection_action_identifier(:service_catalogs, :service_templates, :order)
+      order_request["href"] = /service_requests/
```

Comment:
can this be replaced by line 400 below with something like:

```
  expect_single_resource_query(order_request.merge("href" => a_string_matching(service_requests_url))) 
```

though I'm not sure if a_string_matching works with that method. maybe just.

```
  expect_single_resource_query(order_request.merge("href" => /service_requests/) )
```
 Thanks